home/import_content_pages.py

```diff
@@ -147,39 +170,29 @@ def create_shadow_content_page_from_row(self, row: "ContentRow") -> None:
             parent=row.parent,
             related_pages=row.related_pages,
         )
+        self.shadow_pages[row.slug] = page
+
+        self.add_message_to_shadow_content_page_from_row(row)
 
         if row.is_whatsapp_message:
-            page.enable_whatsapp = True
             page.whatsapp_title = row.whatsapp_title
             if row.is_whatsapp_template_message:
                 page.is_whatsapp_template = True
                 page.whatsapp_template_name = row.whatsapp_template_name
-            page.whatsapp_body.append(
-                ShadowWhatsappBlock(
-                    message=row.whatsapp_body,
-                    next_prompt=row.next_prompt,
-                )
-            )
```

Comment:
This logic was being duplicated in `add_message_to_shadow_content_page_from_row`, so I've removed it from here and used that instead.